Palladium Rifts by Grinning Gecko/src/partials/armor.html

```diff
@@ -1,6 +1,7 @@
 <label
   ><input type="checkbox" class="is-active" name="attr_is_active" value="1"
 /></label>
+<input type="number" placeholder="Movement Penalty" name="attr_mvmt" />
```

Comment:
```suggestion
<input type="number" placeholder="Movement Penalty" name="attr_movementpenalty" />
```
I like to be verbose, especially in here where there's so many similarly named attributes.